# Generated by Django 2.2.2 on 2019-07-01 06:39

from django.db import migrations, models
import django.utils.timezone


class Migration(migrations.Migration):

    dependencies = [
        ('fault_submission', '0002_fault_followed_up'),
    ]

    operations = [
        migrations.AddField(
            model_name='fault',
            name='date_of_creation',
            field=models.DateTimeField(blank=True, default=django.utils.timezone.now),
        ),
    ]
